sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluate/_eval_run.py

```diff
@@ -491,6 +492,29 @@ def log_artifact(self, artifact_folder: str, artifact_name: str = EVALUATION_ART
         if response.status_code != 200:
             self._log_warning("register artifact", response)
 
+        # register artifacts for images if exists in image folder
```

Comment:
Why is image-specific logic included within a generic `log_artifact` function? Would it be possible to call `log_artifact` to upload image artifacts in the upper layer?

Reply:
valid point. lmtal what best we can do.

Reply:
Synced with Waqas offline. Waqas will create a work item and address this in a follow up.